packages/security/src/types/VOGroup.ts

```diff
@@ -0,0 +1,4 @@
+export type VOGroup = {
```

Comment:
what does 'VO' stand for ?

Reply:
yes, we definitely need more docs :)

Reply:
It stands for Virtual Organisation, a remotely managed group of people